content/source/layouts/enterprise2.erb

```diff
@@ -213,6 +213,9 @@
           <li<%= sidebar_current("docs-enterprise2-private-config") %>>
             <a href="/docs/enterprise/private/config.html">Configuration</a>
           </li>
+          <li<%= sidebar_current("docs-enterprise2-private-vault") %>>
+            <a href="/docs/enterprise/private/vault.html">Customer Vault (Installer Beta)</a>
+          </li>
           <li<%= sidebar_current("docs-enterprise2-private-reliability-availability") %>>
             <a href="/docs/enterprise/private/reliability-availability.html">Reliability & Availability</a>
```

Comment:
Unrelated, but I think this `&` should be `&amp;`.

Reply:
huh?

Reply:
`Reliability & Availability` → `Reliability &amp; Availability`

Reply:
Ah. Feel free to open another PR to fix that. 😄